### PR TITLE
Update governing docs and update README on how to do so.

### PR DESCRIPTION
**Pull request by akath20**

### CODE REVIEW

**Comment by akath20** [comment]
The file changes in this PR are already live on the website. Need to figure out how/why this happened.

**Comment by Tjzabel** [comment]
@akath20 what do you mean?

**Comment by Tjzabel** [comment]
@akath20 The submodule governing docs changes are on the master branch of the governing docs repo.

To me, it looks like you just updated the submodule to be reflected here in this repository.

**Comment by Tjzabel** [comment]
@akath20 what I mean is that those changes I see in the submodule are already present in the [master branch](https://github.com/rit-sse/governing-docs/blob/master/constitution.md)/ of the governing docs.
That's why I was saying it looks like you are just updating this repository to reflect those changes.

**Comment by Tjzabel** [comment]
Ahhh, I see what you mean. 

sse.rit.edu is pulling from the master branch of the governing docs, when it should be pulling from what's here?

File: app/js/home/components/Hero.jsx [hunk 1]
```diff
@@ -33,8 +33,8 @@ class Hero extends React.Component {
           <div className="fancy-hero-container" />
           <div className="hero-content">
             <h3>Weekly Meetings</h3>
-            <h6>Th @ 4pm</h6>
-            <h6>GOL-1610</h6>
+            <h6>Mondays @ 1:15pm</h6>
+            <h6>GOL-1670</h6>
             <h6>All Are Welcome!</h6>
```
Comment (7imbrook) [inline app/js/home/components/Hero.jsx]: I feel like this should just be part of some configuration management and read at runtime

**Comment by vuzDylan** [comment]
Could make it markdown site already supports rendering markdown

File: app/js/home/components/Hero.jsx (hunk 1 above)
Comment (7imbrook) [reply app/js/home/components/Hero.jsx]: any good configuration provider will do this as well, without needing to redeploy if the app can support it.

Comment (billdybas) [reply app/js/home/components/Hero.jsx]: Sure, but that's not how things are currently setup. Feels unnecessarily complex to add new infrastructure to support changing a date. 😉

**Pull request merged**